shallow_backup/utils.py

```diff
@@ -12,24 +12,36 @@ def run_cmd(command):
 	"""
 	try:
 		if not isinstance(command, list):
-			process = sp.run(command.split(), stdout=sp.PIPE)
+			process = sp.run(command.split(), stdout=sp.PIPE, stderr=sp.DEVNULL)
 			return process
 		else:
-			process = sp.run(command, stdout=sp.PIPE)
+			process = sp.run(command, stdout=sp.PIPE, stderr=sp.DEVNULL)
 			return process
 	except FileNotFoundError:  # If package manager is missing
 		return None
 
 
-def run_cmd_write_stdout(command, filepath):
+def run_cmd_write_stdout(command, filepath, package):
 	"""
 	Runs a command and then writes its stdout to a file
 	:param: command str representing command to run
+	:param: filepath str file to write command's stdout to
+	:param: package str name of package to print for failed commands
 	"""
 	process = run_cmd(command)
-	if process:
+	if process and process.returncode == 0:
 		with open(filepath, "w+") as f:
 			f.write(process.stdout.decode('utf-8'))
+		return 0
```

Comment:
Were you planning on doing something with the return value? I couldn't find an instance of the return value being checked in the PR.

Reply:
uh apparently I use it on line 123 in backup.py
```
if run_cmd_write_stdout(command, temp_file_path, 'npm') == 0:
```
I think I can just turn it into
```
if not run_cmd_write_stdout(command, temp_file_path, 'npm'):
```
and get rid of the `return 0` if you want but I thought it was more clear with the 0 code since you can tell it's more of a shell command return code check rather than just a not something check.

Reply:
although `None` is `0` now that I think about it, imma get rid of the return 0

Reply:
> uh apparently I use it on line 123 in backup.py

I missed this, my b.

> although None is 0 now that I think about it, imma get rid of the return 0

Works for me.